packages/paste-core/components/summary-detail/src/SummaryToggleButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, type BoxProps } from "@twilio-paste/box";
+import { Button } from "@twilio-paste/button";
+import { ChevronDisclosureIcon } from "@twilio-paste/icons/esm/ChevronDisclosureIcon";
+import * as React from "react";
+
+interface SummaryToggleButtonProps {
+  element?: BoxProps["element"];
```

Comment:
observation: I'm sure we have this else where, but it's the first time I recall noticing it. This component is a "private" component to the package. As a consumer you don't need to know about it... _except_ for the customization element name. As we don't need to export it, we don't document its API... but there is an API just for customization.

We don't need to do anything with that information, I'm just saying it out loud becuase I don't personally know what to do with that information right now

Reply:
Fixed with a refactor to make this exposes and composable